Description: enable building info documentation, remove incompatible command line arg
Author: Dan Weatherill <dan.weatherill@cantab.net>
Last-Update: 2017-10-31


Index: gmsh-3.0.5-source/CMakeLists.txt
===================================================================
--- gmsh-3.0.5-source.orig/CMakeLists.txt
+++ gmsh-3.0.5-source/CMakeLists.txt
@@ -1736,8 +1736,8 @@ if(MAKEINFO)
   add_custom_target(txt DEPENDS ${TEX_DIR}/gmsh.txt)
   add_custom_command(OUTPUT ${TEX_DIR}/gmsh.html DEPENDS ${TEX_SRC}
     COMMAND ${MAKEINFO} --html --css-ref=http://gmsh.info/gmsh.css
-    --no-split --set-customization-variable
-    EXTRA_HEAD='<meta name="viewport" content="width=device-width,initial-scale=1.0">'
+    --no-split 
+    -D EXTRA_HEAD='<meta name="viewport" content="width=device-width,initial-scale=1.0">'
     ARGS ${TEX_DIR}/gmsh.texi WORKING_DIRECTORY ${TEX_DIR})
   add_custom_target(html DEPENDS ${TEX_DIR}/gmsh.html)
   install(FILES ${TEX_DIR}/gmsh.html DESTINATION ${GMSH_DOC} OPTIONAL)
@@ -1748,7 +1748,7 @@ endif(MAKEINFO)
 find_program(TEXI2PDF texi2pdf)
 if(TEXI2PDF)
   add_custom_command(OUTPUT ${TEX_DIR}/gmsh.pdf DEPENDS ${TEX_SRC}
-                     COMMAND ${TEXI2PDF} ARGS gmsh.texi
+                     COMMAND ${TEXI2PDF} ARGS ${TEX_DIR}/gmsh.texi
                      WORKING_DIRECTORY ${TEX_DIR})
   add_custom_target(pdf DEPENDS ${TEX_DIR}/gmsh.pdf)
   install(FILES ${TEX_DIR}/gmsh.pdf DESTINATION ${GMSH_DOC} OPTIONAL)
